lib/rules/id-length.js

```diff
@@ -40,6 +42,10 @@ module.exports = function(context) {
             return parent.id === node;
         },
         "Property": function(parent, node) {
+            if (!properties) {
```

Comment:
>Edit: Your last sentence wasn't in the original message! :smile:

Yeah, I added that just after. That's my preference but I can't enforce you to do anything :) We still need someone from @eslint/eslint-team to review and sign this off ;)